.circleci/config.yml

```diff
@@ -720,42 +720,8 @@ workflows:
       not:
         equal: [ scheduled_pipeline, << pipeline.trigger_source >> ]
     jobs:
-      - lint:
```

Comment:
all of these are coming back before merging, right?

Reply:
Removed commit 👍🏻  was making sure it was still working.